current/runtime/floats.md

```diff
@@ -4,9 +4,11 @@ lang: en
 title: Floating Point Numbers
 ---
 
-When developing for the Substrate runtime and doing calculations it is appealing to use floating point numbers for e.g. representing fractions. This document covers why traditional floating point numbers are problematic in a runtime context and what alternatives exist.
+When developing for the Substrate runtime and doing calculations it is appealing to use floating point numbers for e.g. representing fractions. In Substrate it is important that each node arrives at the same deterministic result for consensus. This document covers why traditional floating point numbers are problematic in runtime and what alternatives exist.
 
-## Problems with Floating Point Numbers in the Runtime
+## Problems with Floating Point Numbers in Runtime
```

Comment:
Curious why you opted for dropping the "the". AFAIK we use runtime generically and not as Named Thing (i.e. Runtime)? Oh or are you doing this because you use like "in context"? Like "We don't use non-deterministic operations in runtime [development]"?

Reply:
yes-- using this in context- if you prefer generically calling it-- imo works both ways

Reply:
I think I've only seen it used generically while working at Parity (which hasn't been that long ^^) and would personally make it explicit (i.e. use "runtime development") when using it as an activity.
But in the end I only care wheter it's consistent (over our communication and docs). I could get used to your variant *shrug*